tests/4.5/target_teams_distribute_parallel_for/test_target_teams_distribute_parallel_for_default.c

```diff
@@ -0,0 +1,83 @@
+//===----  test_target_teams_distribute_parallel_for_default.c- combined consutrct -===//
+// 
+// OpenMP API Version 4.5 Nov 2015
+// 
+// testing the combined construct target teams distribute parallel for
+//
+//===---------------------------------------------------------------------===//
+
+#include <omp.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include "ompvv.h"
+
+
+int DefaultFirstPrivate() {
+  int ErrCount = 0;
+  int Arr[32];
+  for (int i = 0; i < 32; ++i) {
+    Arr[i] = i;
+  }
+
+#pragma omp target teams distribute parallel for num_teams(2) thread_limit(10)\
+        default(firstprivate)
+  for (int i = 0; i < 32; ++i) {
+    if (Arr[i] != i) {
+#pragma omp atomic
+      ErrCount += 1;
```

Comment:
ErrCount should be both shared and mapped tofrom to get the correct values on the host.

Reply:
mapped ErrCount tofrom using map()